### ISSUE TITLE
Retrieve list of Items from Skritter

### ISSUE THREAD

**Issue by billglover** [billglover/character-stats#1]
Before we can analyse characters that a user finds difficult we need to retrieve the list of Characters that a user has studied. Skritter refers to these as Items.

> The Item endpoint gives you complete access to what words a user is studying, how well they know it, and some key info about the history of an item's study. Use this endpoint if you would like to know:
> 
> * What words are being studied.
> * How well any given word/part is known.
> * What was studied last, or will be studied next.
– [Skritter API Docs](https://www.skritter.com/api/v0/docs/endpoints/items)

To retrieve a list of items, we need to make a call to the following API endpoint:
```
http://legacy.skritter.com/api/v0/items
```

Some query parameters worth considering are:
* `cursor` - for pagination
* `include_vocabs` - to include the vocab items themselves
* `include_heisigs` - to include heist definitions (Chinese only)
* `include_top_mnemonics` - unclear if this includes user mnemonic

This issue is to retrieve the Vocab items. A separate issue will be raised for storing them.
